helmfile/charts/node-local-dns/templates/node-local-dns.yaml

```diff
@@ -17,6 +17,7 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: node-local-dns
+  namespace: kube-system
```

Comment:
You can delete the namespace in all files. It should instead be added by helm during the installation.
```suggestion
```